### PR TITLE
-adding some docs for conflict resolution and proposition tracking

### PR DESCRIPTION
**Pull request by sbenedicadb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/sources/shared/event-tracking.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+                "eventType" : "personalization.request"
+            }
+        }
+    ]
+}
+```
+
+### Proposition is triggered - decisioning.propositionTrigger
+
+An event similar to the following is emitted any a user has met the client-side criteria that qualifies them for a proposition:
```
Comment (rymorale) [inline Documentation/sources/shared/event-tracking.md:66]: "emitted any time..."

**Review by rymorale — APPROVED** [review]
just a minor text issue to resolve but looks good to me

File: Documentation/sources/shared/event-tracking.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+}
+```
+
+### Proposition was suppressed - decisioning.propositionSuppressDisplay
+
+This event is emitted any time the triggering criteria are met for a message, but the SDK does not display the message.
+
+The value in the `xdm._experience.decisioning.propositionAction.reason` property is used to describe why the display was suppressed.
+
+#### Use case #1
```
Comment (jfkhoury) [inline Documentation/sources/shared/event-tracking.md:252]: Love the use cases!

**Review by jfkhoury — APPROVED** [review]
LOVE IT!

**Pull request merged**